egs/librispeech/ASR/pruned_transducer_stateless7_ctc/train.py

```diff
@@ -1035,8 +1035,12 @@ def run(rank, world_size, args):
     if world_size > 1:
         logging.info("Using DDP")
         model = DDP(model, device_ids=[rank], find_unused_parameters=True)
-
-    optimizer = ScaledAdam(model.parameters(), lr=params.base_lr, clipping_scale=2.0)
+        
+    parameters_names = []
```

Comment:
Wondering why it did not throw an assertion error before.

@yaozengwei Could you please have a check?

Reply:
I see. I trained this model before the optimizer files changed (see 4fee3e7f1ea6c2aefe7594e325ede1e530e54d3d).